include/fty_common_asset_types.h

```diff
@@ -50,8 +65,9 @@ namespace persist {
         SWITCH,
         STORAGE,
         VIRTUAL,
-        N_A = 11,
-        /* ATTENTION: don't change N_A id. It is used as default value in initdb.sql for types, that don't have N_A */
+
+        VM,
+        N_A,
```

Comment:
you should keep the explicit "N_A = 11" + its belonging comment for now, just to depict the actual situation!